libvmi/driver/memory_cache.c

```diff
@@ -297,6 +314,15 @@ memory_cache_insert(
     }
 }
 
+void memory_cache_remove(
```

Comment:
Ah yes, you're right.  Funny, I was looking for that else and just scanned right over it.  Must be early still ;-)